compiler/rustc_interface/src/tests.rs

```diff
@@ -762,6 +762,7 @@ fn test_unstable_options_tracking_hash() {
     tracked!(coverage_options, CoverageOptions { branch: true });
     tracked!(crate_attr, vec!["abc".to_string()]);
     tracked!(cross_crate_inline_threshold, InliningThreshold::Always);
+    tracked!(debug_fmt_detail, DebugFmtDetail::Full);
```

Comment:
This should **not** be the default, since this test checks that changing the *default* produce a different crate hash.
```suggestion
    tracked!(debug_fmt_detail, DebugFmtDetail::Shallow);
```